corpus-client/src/KeyboardNavigation.elm

```diff
@@ -0,0 +1,30 @@
+module KeyboardNavigation exposing (..)
+
+import Json.Decode as Decode
+
+
+
+
+type Direction
+  = Previous
+  | Next
+
+
+keyDecoder : (Direction -> msg) -> Decode.Decoder msg
+keyDecoder msgForDirection =
+  let
+    maybeDirection =
+      Decode.map2
+      (\key isAltDown ->
+         case (key, isAltDown) of
+           ("j", True) -> Just Next
+           ("k", True) -> Just Previous
+           _ -> Nothing)
```

Comment:
Interestingly, using `"Enter"` here (which would be nice UX) invokes the race condition (already noted in comments);
- `Enter` invokes the `input`'s `onChange` sents a `PATCH` request, the response of which updates the selected `Email` data.
- but if `Enter` is used to select the next email, this means it updates the wrong one.

Ideally, the `PATCH` request should be a bit tighter about *what* data it updates; e.g. what the selection was when it sent the request is probably enough?